src/components/axes/timeAxis.ts

```diff
@@ -1,6 +1,17 @@
 ///<reference path="../../reference.ts" />
 
 module Plottable {
+
+export class TimeInterval {
```

Comment:
Are we worried that people can instantiate a `TimeInterval`? Not that there's anything they could do with it, but making `TimeInterval` a `module ` would prevent that right?